raft_test.go

```diff
@@ -2256,15 +1895,9 @@ func TestDisruptiveFollowerPreVote(t *testing.T) {
 	// n1.state == StateLeader
 	// n2.state == StateFollower
 	// n3.state == StateFollower
```

Comment:
This comment is now unnecessary - the code below is self-explanatory. Remove it?

Same in a few cases below and above.